website/content/docs/release-notes/consul/v1_17_x.mdx

```diff
@@ -15,14 +15,10 @@ We are pleased to announce the following Consul updates.
 
   These APIs are the foundation for future versions of Consul and enable new functionalities, such as multi-port and host-name-based canary routing and routing traffic through headless services in native Kubernetes deployments.
 
-  For more information, refer to the [Catalog API v2](/consul/docs/k8s/multiport#catalog-api-v2-beta) section in the documentation.
-
   <Note> These APIs are in beta and under active development, so we do not recommend using them in production. </Note>
 
 - **Multi-port services in Consul:** You can now register services with multiple ports per service. The v2 catalog API enables a single sidecar proxy to support workloads on different ports. This significantly reduces the operational overhead for managing Consul service mesh. Support for other runtimes outside of Kubernetes is planned for future releases of Consul.
 
-  Refer to the [Multi-port services for service mesh](/consul/docs/k8s/multiport#catalog-api-v2-beta) and [Configure multi-port services](/consul/docs/k8s/multiport/configure) for more information.
```

Comment:
~ Given these are part of release notes, we could instead link to the versioned copies, assuming we don't intend to backport these deletions to versioned docs:
https://developer.hashicorp.com/consul/docs/v1.17.x/k8s/multiport
https://developer.hashicorp.com/consul/docs/v1.17.x/k8s/multiport/configure

Reply:
Ahh, that's a much better idea! My first attempt was pure laziness.